cli/things_test.go

```diff
@@ -28,11 +28,15 @@ var (
 	tokenWithoutDomain = "valid"
 	relation           = "administrator"
 	all                = "all"
+	get                = "get"
 )
 
 var thing = mgsdk.Thing{
-	ID:       testsutil.GenerateUUID(&testing.T{}),
-	Name:     "testthing",
+	ID:   testsutil.GenerateUUID(&testing.T{}),
+	Name: "testthing",
```

Comment:
```suggestion
	Name: "testThing",
```

Reply:
This is not a variable name but a name of a thing.